### ISSUE TITLE
How I got NMSSaveEditor working in Ubuntu Linux (Feature Request)

### ISSUE THREAD

**Issue by genestrashcan** [goatfungus/NMSSaveEditor#1123]
I was able to get NMSSaveEditor working in Kubuntu 24.04 by running a shell script with the following code below in the directory of the NMSSaveEditor.jar file after i extracted the self extracting archive with WineHQ.

The following below is the code of the shell script i created.

`#!/bin/sh
java -jar NMSSaveEditor.jar -autoupdate `

See if you can create a Linux App Image that will execute a shell script with this code above in the directory of the Jar file.

**Comment by smgrey** [comment]
You can just double click the jar file and it will execute. Linux runs Java normally, why would there be a need for an app image?

**Comment by wkitty42** [comment]
yeah, no doubt! no need for an appimage... putting those two lines in a script file will work quite well without all the additional overhead...

**Comment by LockNessMonster1** [comment]
@genestrashcan make sure you dowload the zip, not the exe: https://github.com/goatfungus/NMSSaveEditor/raw/master/NMSSaveEditor.zip (although it shouldn't make a difference but it's easier to extract a zip than an exe on Linus :) )

After download extract and make sure you only run `java -jar NMSSaveEditor.jar`. it will run, not sure if it will find the save dir like in Windows since I don't run NMS on Linux, or even if it will work properly as it does on Windows, but it will execute.

So be careful since this is most likely untested and you might run into issues. Not trying to discourage though, I'm an avid Linux user myself :) 

Just not brave enough to run NMS on Linux :D
